app/uk/gov/hmrc/servicecommissioningstatus/model/Environment.scala

```diff
@@ -18,7 +18,7 @@ package uk.gov.hmrc.servicecommissioningstatus.model
 
 import play.api.libs.json.{Format, JsError, JsResult, JsString, JsSuccess, JsValue}
 
-sealed trait Environment { def asString: String }
+sealed trait Environment { def asString: String; override def toString = asString }
```

Comment:
The reason for introducing `asString`, rather than overriding `toString` is to be explicit about the type conversion, and leave `toString` as defined by Java Object - as an aid for debugging (hiding the fact that the type is `Environment` and not `String` can cause problems when debugging/investigating unit test failures for example).

Reply:
Thanks changed